python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -75,23 +77,75 @@ def _build_fil_classifier(m, data, arg={}, tmpdir=None):
         "silent": 1, "eval_metric": "error",
         "objective": "binary:logistic", "tree_method": "gpu_hist",
     }
-    params.update(arg)
-    max_depth = arg["max_depth"]
-    num_rounds = arg["num_rounds"]
+    params.update(args)
+    max_depth = args["max_depth"]
+    num_rounds = args["num_rounds"]
     n_feature = data[0].shape[1]
     train_size = data[0].shape[0]
     model_name = f"xgb_{max_depth}_{num_rounds}_{n_feature}_{train_size}.model"
     model_path = os.path.join(tmpdir, model_name)
     bst = xgb.train(params, dtrain, num_rounds)
     bst.save_model(model_path)
 
-    return m.load(model_path, algo=arg["fil_algo"],
-                  output_class=arg["output_class"],
-                  threshold=arg["threshold"],
-                  storage_type=arg["storage_type"])
+    return m.load(model_path, algo=args["fil_algo"],
+                  output_class=args["output_class"],
+                  threshold=args["threshold"],
+                  storage_type=args["storage_type"])
 
 
-def _build_treelite_classifier(m, data, arg={}, tmpdir=None):
+def _build_fil_skl_classifier(m, data, args, tmpdir):
+    """Trains an SKLearn classifier and returns a FIL version of it"""
+
+    train_data, train_label = _training_data_to_numpy(data[0], data[1])
+
+    params = {
+        "n_estimators": 10,
```

Comment:
here too - would be better to default to n_estimators=100, the current default in 0.22 (a change from the old sklearn 0.21 defaults)

Reply:
Done.